src/convert/utils/configurable-type-provider.ts

```diff
@@ -21,18 +22,21 @@ export class ConfigurableTypeProvider {
       // Record<any, any>
       return t.tsTypeReference(
         t.identifier("Record"),
-        t.tsTypeParameterInstantiation([t.tsAnyKeyword(), t.tsAnyKeyword()])
+        t.tsTypeParameterInstantiation([
+          FlowCompatTypes.any,
+          FlowCompatTypes.any,
+        ])
       );
     } else {
-      return t.tsAnyKeyword();
```

Comment:
Basically, looked for `t.tsAnyKeyword()` and replaced if with `FlowCompatTypes.any`